docs/Plugins/jira.md

```diff
@@ -7,129 +7,122 @@ description: >
+## Data Refresh Policy
 
-## Configuration
-Configuring Jira via [config-ui](/UserManuals/ConfigUI/Jira.md).
-
-## Collect Data From JIRA
-
-To collect data, select `Advanced Mode` on the `Create Pipeline Run` page and paste a JSON config like the following:
-
-> <font color="#ED6A45">Warning: Data collection only supports single-task execution, and the results of concurrent multi-task execution may not meet expectations.</font>
+Check out the [data refresh policy](/Overview/SupportedDataSources.md#jira) of this plugin.
 
-```
-[
-  [
-    {
-      "plugin": "jira",
-      "options": {
-          "connectionId": 1,
-          "boardId": 8,
-          "since": "2006-01-02T15:04:05Z"
-      }
-    }
-  ]
-]
-```
+## Metrics
 
-- `connectionId`: The `ID` field from **JIRA Integration** page.
-- `boardId`: JIRA board id, see "Find Board Id" for details.
-- `since`: optional, download data since a specified date only.
+Metrics that can be calculated based on the data collected from Jira:
 
+- [Requirement Count](/Metrics/RequirementCount.md)
+- [Requirement Lead Time](/Metrics/RequirementLeadTime.md)
+- [Requirement Delivery Rate](/Metrics/RequirementDeliveryRate.md)
+- [Requirement Granularity](/Metrics/RequirementGranularity.md)
+- [Bug Count](/Metrics/BuildCount.md)
+- [Bug Age](/Metrics/BugAge.md)
+- [Bugs Count per 1k Lines of Code](/Metrics/BugCountPer1kLinesOfCode.md)
+- [Incident Age](/Metrics/IncidentAge.md)
+- [Incident Count per 1k Lines of Code](/Metrics/IncidentCountPer1kLinesOfCode.md)
 
-## API
 
-### Data Connections
+## Configuration
+Configuring Jira via [config-ui](/UserManuals/ConfigUI/Jira.md).
 
-1. Get all data connection
+## API Sample Request
+
+### Create a new data connection
+
+```shell
+curl -X 'POST' \
+  'http://127.0.0.1:8080/plugins/jira/connections' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "jira test",
+  "endpoint": "https://merico.atlassian.net/rest/",
+  "proxy": "http://127.0.1.1:1080",
+  "username": "your user name",
+  "password": "your password"
+}'
+```
 
-```GET /plugins/jira/connections
-[
-  {
-    "ID": 14,
-    "CreatedAt": "2021-10-11T11:49:19.029Z",
-    "UpdatedAt": "2021-10-11T11:49:19.029Z",
-    "name": "test-jira-connection",
-    "endpoint": "https://merico.atlassian.net/rest",
-    "basicAuthEncoded": "basicAuth",
-    "epicKeyField": "epicKeyField",
-      "storyPointField": "storyPointField"
+### Create a new transformation rule
+
+```shell
+curl -X 'POST' \
+  'http://127.0.0.1:8080/plugins/jira/transformation_rules' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "epicKeyField": "",
+  "storyPointField": "",
+  "remotelinkCommitShaPattern": "",
+  "typeMappings": {
+    "10040": {
+      "standardType": "Incident",
+      "statusMappings": null
+    }
   }
-]
+}'
 ```
 
-2. Create a new data connection
+### Create a new scope
```

Comment:
What kind of users will use this API? If users rarely use this, can we remove it?

Reply:
you are right, I will remove them